test/test.index.ts

```diff
@@ -443,7 +442,7 @@ describe('linkinator', () => {
     });
     assert.ok(!results.passed);
     const err = results.links[0].failureDetails![0] as Error;
-    assert.ok(/Nock: Disallowed net connect for/.test(err.message));
```

Comment:
Could I trouble you to swap this to a `assert.match` (as we talked about in a different PR)?

Reply:
From a quick look, I'd need to use chai for that, right? If so, then I get TS errors:

>Property 'rejects' does not exist on type 'AssertStatic'

Reply:
aye, the import needs to be destructured - `import {assert} from 'chai';`